### ISSUE TITLE
Modifier.clickable doesn't work

### ISSUE THREAD

**Issue by jsericksk** [ArnyminerZ/MarkdownText#45]
When using ```Modifier.clickable``` in MarkdownText it doesn't work, the click is not detected. It also didn't work with ```Modifier.combinedClickable```.

```kotlin
MarkdownText(
    markdown = "Text **bold**",
    color = Color.White,
    fontSize = 24.sp,
    fontWeight = FontWeight.Bold,
    modifier = Modifier.clickable { Log.d("MarkdownText", "Text onClick")},
)
```

**Comment by ArnyminerZ** [comment]
Hi, the component implements a custom click detector for handling links annotated in Markdown. However, I can add a `onClick` callback or something if you need it.

**Comment by ArnyminerZ** [comment]
This has been implemented in #46, please take a look. Will be released with MarkdownText 1.2.4

**Comment by jsericksk** [comment]
Wow! That was really fast! Thank you very much for the quick response and for the library. Worked perfectly. :)
